src/enclave/outside/attestation/AzureAttestationServiceClient.cpp

```diff
@@ -0,0 +1,250 @@
+#include <enclave/outside/attestation/AzureAttestationServiceClient.h>
```

Comment:
To read this file it may be helpful to keep in mind that `validateQuote` in `src/enclave/outside/attestation/AzureAttestationServiceClient.cpp` calls the methods in this file in this order:

```cpp
    AzureAttestationServiceClient client(attestationProviderUrl);

    // Send enclave quote to remote attestation service for validation
    std::string jwtResponse = client.attestEnclave(enclaveInfo);

    // Validate JWT response token
    client.validateJwtToken(jwtResponse);
```